payments-core/src/test/java/com/stripe/android/view/CardInputWidgetTest.kt

```diff
@@ -1742,6 +1742,7 @@ internal class CardInputWidgetTest {
             expiryDateEditText.append("12")
             expiryDateEditText.append("50")
             cvcEditText.append("123")
+            cardNumberEditText.requestFocus()
```

Comment:
Fixing this test actually brought up a UX experience that I'm not sure if we should look into solving. If the merchant chooses not to display the postal code, then the customer is not shown the CBC dropdown is not shown when focused on the CVC field since the CBC dropdown action is replaced with the CVC logo. The customer has to click on either the card number or expiration date field to re-display it.